adapters/oidc/servlet-filter/pom.xml

```diff
@@ -30,6 +30,16 @@
     <name>Keycloak Servlet Filter Adapter Integration</name>
     <description/>
 
+    <properties>
+        <keycloak.osgi.export>
+            org.keycloak.adapters.servlet.*
+        </keycloak.osgi.export>
+        <keycloak.osgi.import>
+            org.keycloak.*;version="${project.version}",
```

Comment:
Would you add explicit dependency to `javax.servlet` package similarly to [this](https://github.com/AcutusPro/keycloak/blob/42e6337954996275389cf70494a3b313816494e0/adapters/oidc/osgi-adapter/pom.xml#L40)? (Thanks @grgrzybek for pointin this out)